### ISSUE TITLE
Update to TaskBuilder 2.0

### ISSUE THREAD

**Issue by Lanayx** [giraffe-fsharp/Giraffe#292]
I've tried to use latest task builder, but got many exceptions like 
```
An unhandled exception has occurred while executing the request.
System.MissingMethodException: Method not found: 'TaskBuilder FSharp.Control.Tasks.ContextInsensitive.get_task()'.
   at Giraffe.Middleware.Invoke@39-1.Invoke(FSharpOption`1 _arg1)
   at FSharp.Control.Tasks.TaskBuilder.StepStateMachine`1.nextAwaitable() in D:\src\TaskBuilder\TaskBuilder.fs:line 38
```
or
```
Unable to cast object of type 'System.Threading.Tasks.UnwrapPromise`1[Microsoft.FSharp.Core.FSharpResult`2[System.String,System.String]]' to type 'System.Threading.Tasks.Task`1[Microsoft.FSharp.Core.FSharpResult`2[System.String,System.Object]]

at FSharp.Control.Tasks.TaskBuilder.run[a](FSharpFunc`2 firstStep)\n--- End of stack trace from previous location where exception was thrown ---\n   at FSharp.Control.Tasks.TaskBuilder.bindTaskConfigureFalse[a,b](Task`1 task, FSharpFunc`2 continuation)\n   at FSharp.Control.Tasks.TaskBuilder.run[a](FSharpFunc`2 firstStep)\n--- End of stack trace from previous location where exception was thrown ---\n   at FSharp.Control.Tasks.TaskBuilder.bindTaskConfigureFalse[a,b](Task`1 task, FSharpFunc`2 continuation)\n   at FSharp.Control.Tasks.TaskBuilder.run[a](FSharpFunc`2 firstStep)\n--- End of stack trace from previous location where exception was thrown ---\n   at FSharp.Control.Tasks.TaskBuilder.tryFinally@196.Invoke(Unit unitVar0)\n   at FSharp.Control.Tasks.TaskBuilder.StepStateMachine`1.nextAwaitable()\n--- End of stack trace from previous location where exception was thrown ---\n   at FSharp.Control.Tasks.TaskBuilder.bindTaskConfigureFalse@131.Invoke(Unit unitVar0)\n   at FSharp.Control.Tasks.TaskBuilder.StepStateMachine`1.nextAwaitable()\n--- End of stack trace from previous location where exception was thrown ---\n   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)\n   at Libs.Http.LogMiddleware.Invoke@25-2.Invoke(Unit unitVar0) in D:\\Work\\Imho\\Api\\src\\Libs\\Http\\LogMiddleware.fs:line 25\n   at FSharp.Control.Tasks.TaskBuilder.tryWith[a](FSharpFunc`2 step, FSharpFunc`2 catch)

```
so currently I have to stick to TaskBuilder 1.2.0-rc version

**Comment by dustinmoris** [comment]
This has been released with [Giraffe 2.0.0](https://github.com/giraffe-fsharp/Giraffe/releases/tag/v2.0.0).

**Comment by Lanayx** [comment]
@dustinmoris  I'm afraid this is not enough... To update to second version, namespaces should be updated to V2 (like FSharp.Control.Tasks.V2.ContextInsensitive). I'm not the fan of this solution, but looks like that we have to follow it.

**Comment by dustinmoris** [comment]
Oh crap, ok yeah thanks for letting me know. I'll push a hotfix later
tonight! Currently away from my desk!

**Comment by Lanayx** [comment]
You are welcome!

**Comment by dustinmoris** [comment]
@Lanayx: Little delay due to this: https://github.com/rspeele/TaskBuilder.fs/issues/23

**Comment by dustinmoris** [comment]
Ok I just published version Giraffe 2.0.1 with the fix. Might take a few minutes until the NuGet package has been indexed.

**Comment by Lanayx** [comment]
Thank you, I'll check it out
